plugins/modules/provision_workflow_manager.py

```diff
@@ -802,23 +836,39 @@ def get_wireless_params(self):
             }
         ]
 
-        if self.compare_dnac_versions(self.get_ccc_version(), "2.3.5.3") <= 0:
-            self.log("Checking for managed AP locations in Catalyst Center version <= 2.3.5.3", "DEBUG")
-            ap_locations = wireless_params[0].get("managedAPLocations")
-
-            if not (ap_locations and isinstance(ap_locations, list)):
-                self.log("Validating AP locations: {0}".format(ap_locations), "DEBUG")
-                msg = "Missing Managed AP Locations: Please specify the intended location(s) for the wireless device \
-                    within the site hierarchy."
-                self.log(msg, "CRITICAL")
-                self.module.fail_json(msg=msg, response=[])
-
-            for ap_loc in self.validated_config.get("managed_ap_locations"):
-                site_type = self.get_site_type(site_name_hierarchy=ap_loc)
-                self.log("Checking site type for AP location '{0}', resolved type: '{1}'".format(ap_loc, site_type), "DEBUG")
-                if site_type != "floor":
-                    self.log("Managed AP Location must be a floor", "CRITICAL")
-                    self.module.fail_json(msg="Managed AP Location must be a floor", response=[])
+        ap_locations = wireless_params[0].get("managedAPLocations")
+        if not (ap_locations and isinstance(ap_locations, list)):
```

Comment:
Make it more readable..

`if not ap_locations or not isinstance(ap_locations, list):
`

Reply:
fixed the code